runtime-interface/src/lib.rs

```diff
@@ -116,21 +92,28 @@ pub trait GearRI {
         (gas_left, res)
     }
 
-    fn get_lazy_pages_status() -> Option<Status> {
-        lazy_pages::get_status()
+    fn lazy_pages_status() -> (Status,) {
```

Comment:
Why not just `Status`?
```suggestion
    fn lazy_pages_status() -> Status {
```